packages/web-components/src/components/cta/video-cta-composite.ts

```diff
@@ -91,7 +91,7 @@ class DDSVideoCTAComposite extends ModalRenderMixin(HostListenerMixin(LitElement
     const { href, videoName: customVideoName, videoDescription } = event.detail;
     (event.target as VideoCTAMixinImpl).videoThumbnailUrl = KalturaPlayerAPI.getThumbnailUrl({
       mediaId: href,
-      width: '320',
+      width: event.target.offsetWidth,
```

Comment:
```suggestion
      width: event?.target?.offsetWidth,
```

it looks like this is causing a test failure since it may be `null`